index.html

```diff
@@ -130,8 +145,12 @@ <h2> Tonic</h2>
             <h2> Multi-Post Stories</h2>
 
             <ul>
-              <li>CANOPY</li>
-              <li>Back End Dev</li>
+              <li>FACEBOOK</li>
+              <li class="bullet">               
+              </li>
+              <li>Full Stack Dev</li>
+              <li class="bullet">               
+              </li>
               <li>2015</li>
```

Comment:
- [ ] Good job on your design 👍🏽 . Kindly ensure that all your project card descriptions match the Figma design. Please add their appropriate colors and font-weight😄

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/199579895-7304ec4a-1cdf-4da4-b6fd-ea5514efe8d7.png)  | ![image](https://user-images.githubusercontent.com/63473584/199579982-1961e43f-6a88-47d3-b50c-88d9b2115bcd.png)  |